internal/schemautil/schemautil.go

```diff
@@ -40,21 +41,6 @@ func OptionalIntPointer(d *schema.ResourceData, key string) *int {
 	return &intValue
 }
 
-// OptionalFloatPointer retrieves a float pointer to a field, if the field is not set, returns nil.
-func OptionalFloatPointer(d *schema.ResourceData, key string) *float64 {
-	val, ok := d.GetOk(key)
-	if !ok {
-		return nil
-	}
-
-	floatValue, ok := val.(float64)
-	if !ok {
-		return nil
-	}
-
-	return &floatValue
-}
-
```

Comment:
🔎 This cleanup was missed in https://github.com/aiven/terraform-provider-aiven/pull/1691, good to get it cleaned up now